components/PartnerDetail/HeroSection.js

```diff
@@ -130,7 +131,17 @@ const HeroSection = ({
             target="blank"
             isLocal={false}
           />
-          <DownArrow icon="arrow" />
+          <DownArrow
+            icon="arrow"
+            onClick={() => {
```

Comment:
Instead of having this scroll to a certain point on the page, you could give the first section an id and then use an `<a>` tag to link to that section along the lines of `href="#partner-about"`. Or whatever id the section is given. That way it will always work across all sizes and styling changes as they get made.